packages/exporter-trace-otlp-http/src/platform/browser/util.ts

```diff
@@ -62,13 +63,19 @@ export function sendWithXhr(
     'Content-Type': 'application/json',
   };
 
+  xhr.timeout = exporterTimeout;
+
   Object.entries({
     ...defaultHeaders,
     ...headers,
   }).forEach(([k, v]) => {
     xhr.setRequestHeader(k, v);
   });
 
+  xhr.ontimeout = function () {
+    xhr.abort();
```

Comment:
is this needed? I'm not an browser expert but reading following at MDN:
```
XMLHttpRequest.timeout
Is an unsigned long representing the number of milliseconds a request can take before automatically being terminated.
```
I interpret this that abort happens automatic and the `ontimeout` handler is just a FYI to log/improve error code,...